contents/docs/product-analytics/paths.mdx

```diff
@@ -144,161 +98,123 @@ Now that we've identified where users are dropping off, let's dig in deeper on t
+- **Set as path end** - This will set this step as the ending point for your path. This is useful when analyzing the ways in which a user ends up at a specific step or learning why users dropped off.
+- **Exclude path item** - This will add an [exclusion](#exclusions) rule for this specific event
+- **View funnel** - This will open up a new [funnel insight](/docs/product-analytics/funnels) with this event already set as the first step. This is useful when analyzing drop-off of a specific step.
+- **Copy path item name** - Copies the event name to your clipboard
 
 ### Viewing recordings and creating cohorts from a step
 
 <FeatureAvailability availability={_frontmatter.availability.features.cohorts} />
 
-One of the most powerful features of the Paths insight is the ability to see all the users who passed through a specific step.
-To do this, we simply click on the number next to the name of step which shows how many users arrived at this point, which will bring up a list of all of these users.
+One of the most powerful features of paths is the ability to see all the users who passed through a specific step.
+To do this, click on the number next to the name of step. This brings up a list of all of these users.
 
 <ProductScreenshot
-    imageLight = {usersLight} 
-    imageDark = {usersDark}
+    imageLight = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716305838/posthog.com/contents/path-users-light.png"} 
+    imageDark = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716305838/posthog.com/contents/path-users-dark.png"}
     classes="rounded"
     alt= "list of users who reached a step"
 />
 
-Next to each user, you can click on the Watch recording button to see a recording of the session where they passed through this step.
-In addition to this, if you hover over the name of a step to bring up the menu we saw in the last section, you'll see you can also click on the numbers showing how many users continued and how many users dropped off to see recordings from each outcome.
-The ability to view the recordings of all the users who dropped off at a certain step and compare them to those who made it through is incredibly useful when diagnosing problems with your funnels.
-
-Beyond just viewing session recordings, you can also use the button in the bottom right to save this group of users as a cohort, and use them in other insights to figure out exactly why they dropped off.
+You can then create a [cohort](https://posthog.com/docs/data/cohorts) of users who passed through the step by clicking the **Save as cohort**.
 
-## Using paths
+If you have enabled [session replays](/docs/session-replay), you can also see recordings of a user sessions for this step by clicking the **View recording** button. This is helpful when diagnosing problems with your funnels.
 
-This next section provides detailed information on all the different components and options of the Paths insight.
+## Configuration options
```

Comment:
Could you document "Number of people on each path", I realize on the surface it looks straightforward, but I don't fully understand its purpose.